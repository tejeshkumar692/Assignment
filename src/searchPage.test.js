import React from 'react'
import { render } from '@testing-library/react';
import SearchPage from './searchPage';

describe('<SearchPage /> spec', () => {
    it('renders the component', () => {
        const container = render(<SearchPage />)
        expect(container.firstChild).toMatchSnapshot()
    })
});
