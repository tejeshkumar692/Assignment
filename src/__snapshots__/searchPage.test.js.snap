// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchPage /> spec renders the component 1`] = `undefined`;
